### DISCUSSION TITLE
.read functionality

### DISCUSSION THREAD

**Discussion by PattonK** [pawelsalawa/sqlitestudio#4473] [General]
I have a sqlite db file which receives updates via a .sql file.  In the instructions, it say to use the following:

Ex) .read RDS_2022.01.3_curated_delta.sql

How would I perform this in SqliteStudio?

**Comment by pawelsalawa** [comment]
Right-click on a database and pick "Execute SQL from file".

**Reply by PattonK** [reply]
Thank you so much.

**Comment by PattonK** [comment]
One more question.  The .sql file had a syntax error with one of the lines.  It was a blessing to have the bypass malformed lines checkbox. 
 Is there any type of syntax validation function for a .sql file prior to loading it?

**Reply by pawelsalawa** [reply]
There's no such feature at the moment.
